tools/interactive/interactivetool_wallace.xml

```diff
@@ -24,15 +24,15 @@
         </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
     </environment_variables>
     <command><![CDATA[
-        ln -s '${infile}' /srv/shiny-server/data/inputdata.txt &&
+        ln -s '${infile}' /import/ &&
         /usr/bin/shiny-server.sh
     ]]>
     </command>
     <inputs>
-        <param name="infile" type="data" format="tabular,txt" label="Tabular in Clarion format"/>
+        <param name="infile" type="data" format="tabular,txt,csv,rdata" label="text oriented (tabular or csv) or RData files"/>
     </inputs>
     <outputs>
-        <!--data name="outfile" format="txt" /-->
+        <data name="outfile" format="tabular,txt,csv,rdata" />
```

Comment:
This IT does not produce yet and output ... isn't it?

Reply:
And yes, this IT produces output ;) as at each RShiny module you can export datasets resulting in computation. But here again I need to test things and commits I hope today